schemas/openshift/terraform-resource-1.yml

```diff
@@ -635,24 +627,44 @@ oneOf:
         required:
         - name
         - default
-        - weights
         oneOf:
         - required:
           - ips
         - required:
           - openshift_service
-    paths:
-      type: object
-      additionalProperties: false
-      properties:
-        read:
-          type: array
-          items:
-            type: string
-        write:
-          type: array
-          items:
-            type: string
+    rules:
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        properties:
+          condition:
+            type: object
+            additionalProperties: false
+            properties:
+              path:
+                type: string
+              methods:
+                type: string
```

Comment:
Shouldn't this one be also an `enum`?

Reply:
This was originally a `string` so I kept it that way. I can change it to an `enum`